pep-0011.txt

```diff
@@ -42,81 +43,87 @@ platform to be considered supported by Python as well as providing a
 procedure to remove code for platforms with few or no Python
```

Comment:
The previous line, which I cannot quote, is
"platform to be considered supported by Python as well as providing a"
Should that be CPython?

Reply:
Yep. I'll fix it shortly.